### ISSUE TITLE
Swagger Not showing Fluent validation requried fields when using System.Text.Json

### ISSUE THREAD

**Issue by tnlthanzeel** [micro-elements/MicroElements.Swashbuckle.FluentValidation#129]
I have an ASP Core 7 API which uses Newtonsoft for serialization. I have fluent validations added to my models and added swagger.

The fluent validations show up on swagger UI when I use Newtonsoft for serialization.

![image](https://user-images.githubusercontent.com/18216790/231367002-13fb11e9-5472-4b9f-a9d3-d9827e9d113d.png)

But I recreated the same project and this time using System.Text.Json, then the fluent validations does not show up on swagger UI when I use System.Text.Jsonfor serialization.

![image](https://user-images.githubusercontent.com/18216790/231367035-92240662-5867-4013-bc0d-2d23f0951a5a.png)

here is the repo using System.Text.Json: https://github.com/tnlthanzeel/Inx.Project.Templates

Any help please

**Comment by petriashev** [comment]
I tried your solution and UserCreateDto works in swagger as expected...

**Comment by tnlthanzeel** [comment]
> I tried your solution and UserCreateDto works in swagger as expected...

Yes, it stared working after i updated to the beta version. But on the stable version it does not show the required fields when using system.text.json but workd for newton soft json. Thats the issue

**Comment by snavarropino** [comment]
Working fine in beta 3

Additional info: I am not using auto validation (AddFluentValidationAutoValidation) because I validate with a MediatR behaviour.  Even in this case is working with minimal api with this simple configuration:

builder.Services.AddFluentValidationRulesToSwagger();
